src/tribol/interface/tribol.cpp

```diff
@@ -331,6 +331,17 @@ void setBinningProximityScale( IndexT cs_id, RealT binning_proximity_scale )
         "Setting binning proximity to less than 2.0 can lead to missed contact pairs.  Resetting to 2.0." );
   }
 
+#ifdef BUILD_REDECOMP
+
+  if ( cs->hasMfemData() ) {
+    auto lor_factor = cs->getMfemMeshData()->GetLORFactor();
+    if ( lor_factor > 0 ) {
+      binning_proximity_scale = binning_proximity_scale * static_cast<RealT>( lor_factor );
```

Comment:
Can you clarify why we are not multiplying the binning proximity scale by the lor factor, but in another routine we divide it by the lor factor. This needs some comments to make this clear.